velox/common/memory/ByteStream.h

```diff
@@ -91,64 +94,29 @@ class OStreamOutputStream : public OutputStream {
   std::ostream* out_;
 };
 
-/// Read-only stream over one or more byte buffers.
+/// Read-only byte input stream interface.
 class ByteInputStream {
```

Comment:
Should we move this base class to the base folder as it has both memory stream subclass `BufferInputStream` and file stream subclass `FileInputStream`?